python/cudf/cudf/core/series.py

```diff
@@ -1306,6 +1301,41 @@ def map(self, arg, na_action=None) -> "Series":
             result = self.apply(arg)
         return result
 
+    def _getitem_preprocessed(
+        self,
+        spec: iu.IndexingSpec,
+    ) -> Union[Self, ScalarLike]:
+        """Get subset of entries given structured data
+
+        Parameters
+        ----------
+        spec:
```

Comment:
Use numpydoc style (and similarly in other docstrings). https://numpydoc.readthedocs.io/en/latest/format.html#parameters
```suggestion
        spec : IndexingSpec
```

Reply:
I can never get this right. TBH, I would prefer to not type the parameters in the docstring (since they can get out of date with the types in the function) and our API docs show the types.